templates/repo/commits_list.tmpl

```diff
@@ -4,8 +4,9 @@
 				<tr>
 					<th class="three wide">{{ctx.Locale.Tr "repo.commits.author"}}</th>
 					<th class="two wide sha">SHA1</th>
-					<th class="nine wide message">{{ctx.Locale.Tr "repo.commits.message"}}</th>
+					<th class="seven wide message">{{ctx.Locale.Tr "repo.commits.message"}}</th>
 					<th class="two wide right aligned">{{ctx.Locale.Tr "repo.commits.date"}}</th>
+					{{if $.FileName}}<th class="two wide right aligned">{{ctx.Locale.Tr "repo.diff.view_file"}}</th>{{end}}
```

Comment:
"two wide" seems a awful lot of empty space for a single button. Does `one wide` work?